mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -42,6 +42,10 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 		ARG 12 spawners
 		ARG 13 shouldTickTime
 	METHOD method_14166 spawnParticles (Lnet/minecraft/class_3222;Lnet/minecraft/class_2394;ZDDDIDDDD)Z
+		COMMENT Spawns a particle visible to {@code viewer}, if the viewer is near the provided
+		COMMENT coordinates. Returns true if successful.
```

Comment:
This should be documented using a `@return` tag and explain what constitutes a success.

Reply:
```suggestion
		COMMENT coordinates.
		COMMENT
		COMMENT @return whether the particle packet was sent
```